parameters/world_modeling/world_model_restaurant.yaml

```diff
@@ -30,18 +33,21 @@ plugins:
       min_cluster_size: 0.2
       max_cluster_size: 1.0
       max_gap_size: 10
+
 - name: entity_clearer
   lib: libed_clearer_plugin.so
   enabled: 1
   parameters:
       entity_timeout: 1.0
+
 - name: kinect_integration
   lib: libed_kinect_plugin.so
   enabled: 1
   parameters:
       topic: /amigo/top_kinect/rgbd
-- name: navigation
```

Comment:
Why don't we need the navigation plugin anymore? What happens if we swipe objects into the world model? This might be convenient for future use (although this might be updated then).